app/build.gradle

```diff
@@ -21,8 +21,15 @@ android {
         minSdk 32 // leave at 32 for robolectric tests
         targetSdk rootProject.ext.targetSdk
         versionNameSuffix "-$gitDescribe"
-        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunner "com.stevesoltys.seedvault.KoinInstrumentationTestRunner"
```

Comment:
Why do we need this? We had instrumentation tests before and they worked, right?

Reply:
Yeah, this is [to add mockk spies](https://github.com/seedvault-app/seedvault/blob/259ec52054cba4cd5bdbd35832717266829426e6/app/src/androidTest/java/com/stevesoltys/seedvault/KoinInstrumentationTestApp.kt) for some dependencies.